.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+   extends: [
+      '@ni/eslint-config/typescript',
+      '@ni/eslint-config/typescript-requiring-type-checking',
+      'plugin:prettier/recommended'
+   ],
+   parserOptions: {
+      project: 'tsconfig.json'
+   },
+   rules: {
+      'no-console': 'off',
```

Comment:
I just found it very tedious to get a compile error whenever I do a quick debug output. I wonder what else can we do to allow it during development?

Reply:
It shouldn't cause a compile error. If it's a compile error, is it triggered by the TypeScript compiler? It shouldn't block the use of console for debug outputs.

Reply:
yes what I get is a problem warning because of "tsc -watch -p ./":
![image](https://user-images.githubusercontent.com/79531/120171223-1ac87500-c202-11eb-8b4b-e0a3e4fd85ab.png)

I don't want to remember my choice because it makes sense to stop debugging if there are still open problems.

Reply:
What's the output when you run `tsc -p ./` locally from the command line? Does it indicate `console.log` statements as errors?

Reply:
No this is not a problem. Apparently, it's only the watcher.

Reply:
I will check the watcher later and decide if we let the rule inactive.